src/main/java/io/vertx/redis/client/impl/RedisStandaloneConnection.java

```diff
@@ -180,6 +180,22 @@ private void taintCheck(CommandImpl cmd) {
 
   @Override
   public Future<Response> send(final Request request) {
+    if (context == vertx.getContext()) {
```

Comment:
I think instead we should use `ContextInternal.execute(Handler<Void>)` and modify `doSend` to use a `Promise<Response>` and have something like:

```java
public Future<Response> send(Request request) {
  Promise<Response> promise = vertx.getOrCreateContext().promise();
  context.execute(() -> doSend(request, promise));
  return promise.future();
}

private void doSend(Request request, Promise<Void>) { ... }
```

Reply:
OK, that makes sense. Will fix.

Reply:
Done.